tests/Doctrine/Tests/DBAL/Schema/ColumnTest.php

```diff
@@ -55,6 +56,15 @@ public function testToArray()
         self::assertEquals($expected, $this->createColumn()->toArray());
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation The "unknown_option" option is not supported, setting it is deprecated and will cause an error in 3.0
+     */
+    public function testSettingUnknownOptionIsStillSupported() : void
```

Comment:
@greg0ire, in some cases, [this test is reported as risky](https://ci.appveyor.com/project/doctrine/dbal/build/1.0.81/job/utt2h7bq2p2b5jj7#L239):

> There was 1 risky test:
> 1) Doctrine\Tests\DBAL\Schema\ColumnTest::testSettingUnknownOptionIsStillSupported
> This test did not perform any assertions
> ...
> Legacy deprecation notices (1)

Is it expected?

Reply:
Older versions of Symfony's phpunit bridge do not count expected deprecation as assertion (and thus mark the test risky).

I would advice to always run all test jobs with the latest version of the bridge

Reply:
@wouterj could you suggest the proper `composer.json` constraints? Currently, we have:

https://github.com/doctrine/dbal/blob/46069b6bc84b0f41733a23d3b56e2774d0502c94/composer.json#L24

Reply:
Dropping the left part of the pipe is completely fine IMO.

Reply:
@greg0ire on the build in question, this constraint [was resolved to `v4.0.6`](https://ci.appveyor.com/project/doctrine/dbal/build/1.0.81/job/utt2h7bq2p2b5jj7#L105).

Reply:
Ah sorry I misread this as 4.0.6 fixes the issue… latest is 4.0.7

Reply:
Can we somehow isolate the issue and report it to upstream?

Reply:
I'm trying to reproduce it as we speak but I'm afraid it only ever happens on windows. Should I create a PR with zero tests but the faulty one?

Reply:
It's fine if it helps to isolate the issue.

Reply:
Ok then. See https://github.com/symfony/symfony/issues/26809